nshlib/nsh.h

```diff
@@ -715,6 +715,25 @@ struct nsh_parser_s
 #endif
 };
 
+#ifdef CONFIG_NSH_ALIAS
+struct nsh_alias_s
+{
+  FAR struct nsh_alias_s *next;    /* Single link list for traversing */
+  FAR char               *name;    /* Name of the alias */
+  FAR char               *value;   /* Value behind the name */
+  union
+  {
+    struct
+    {
+      uint8_t             exp : 1; /* Already expanded ? */
+      uint8_t             rem : 1; /* Marked for deletion */
+    };
+
+    uint8_t               flags;   /* Raw value */
```

Comment:
Do we need raw value? Where is it used?

Reply:
I put it there just for completeness, if for whatever reason the flags need to be cleared as a bunch.

Reply:
Yes. For clearing it makes sense. I was just worried about LE vs BE bit positioning, so just want to make sure that fields are not used as `& flags`.

Reply:
Yes, like you said accessing the bitfield via any other way besides the named fields is meaningless. To be precise, it is not even about LE/BE, the relative order of the bits can also change between implementations. Basically the standard says nothing about how the storage of bitfields must be implemented, the compiler can decide everything.